src/test/java/com/indiraactive/fulfillmentplatform/service/SchedulerImplTests.java

```diff
@@ -0,0 +1,23 @@
+package com.indiraactive.fulfillmentplatform.service;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class SchedulerImplTests {
+    @Autowired
+    private SchedulerImpl scheduler;
+
+    @Test
+    public void runsTask() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/app/indira-active/fulfillment-platform/pullRequest?prid=1507512)